@xen-orchestra/backups/_runners/_vmRunners/_AbstractXapi.mjs

```diff
@@ -219,6 +219,13 @@ export const AbstractXapi = class AbstractXapiVmBackupRunner extends Abstract {
     await xapi.barrier()
     // ensure cached object are up to date
     this._jobSnapshotVdis = this._jobSnapshotVdis.map(vdi => xapi.getObject(vdi.$ref))
+
+    // get the datetime of the most recent snapshot
+    const datetimes = this._jobSnapshotVdis.map(({ other_config }) => other_config[DATETIME])
+    datetimes.sort()
+    const lastSnapshotDateTime = datetimes.pop()
```

Comment:
What do you think about inlining this?
This is just to avoid confusion with `datetimes` defined a few lines below.

```suggestion
    const lastSnapshotDateTime = this._jobSnapshotVdis
      .map(({ other_config }) => other_config[DATETIME])
      .sort()
      .pop()
```

Reply:
good idea, and also there is another datetimes array later, that change will limit confusion